### DISCUSSION TITLE
paddlespeech有现成的模型能支持中英文混合的语音合成吗？

### DISCUSSION THREAD

**Discussion by zillionare** [PaddlePaddle/PaddleSpeech#3524] [Q&A] [answered]
as title.

**Comment by zzabolazza** [comment, marked as answer]
我尝试了以下命令，得到的效果很差。
```
paddlespeech tts \
	--input "Printing, in the only sense with which we are at present concerned, differs from most if not from all the arts and crafts represented in the Exhibition。你可以从中选择各种语音处理工具以及预训练模型，支持语音识别，语音合成，声音分类，声纹识别，标点恢复，语音翻译等多种功能，PaddleSpeech Server模块可帮助用户快速在服务器上部署语音服务。" \
	--output=output.wav \
	-v \
	--lang=mix \
	--device=gpu \
	--am=fastspeech2_male \
	--voc=hifigan_csmsc
```

[Uploading output.zip…]()

**Comment by Samge0** [comment]
# 输入的声学模型（AM）信息
am_info = [
    "speedyspeech_csmsc_onnx-zh",
    "fastspeech2_csmsc_onnx-zh",
    "fastspeech2_ljspeech_onnx-en",
    "fastspeech2_aishell3_onnx-zh",
    "fastspeech2_vctk_onnx-en",
    "fastspeech2_cnndecoder_csmsc_onnx-zh",
    "fastspeech2_male_onnx-zh",
    "fastspeech2_male_onnx-en",
    "fastspeech2_male_onnx-mix",
    "fastspeech2_canton_onnx-canton",
    "fastspeech2_mix_onnx-zh",
    "fastspeech2_mix_onnx-en",
    "fastspeech2_mix_onnx-mix",
]

# 输入的声码器（VOC）信息
voc_info = [
    "mb_melgan_csmsc_onnx-zh",
    "pwgan_csmsc_onnx-zh",
    "pwgan_ljspeech_onnx-en",
    "pwgan_aishell3_onnx-zh",
    "pwgan_vctk_onnx-en",
    "pwgan_male_onnx-zh",
    "pwgan_male_onnx-en",
    "pwgan_male_onnx-mix",
    "pwgan_aishell3_onnx-canton",
    "hifigan_male_onnx-en",
    "hifigan_male_onnx-mix",
    "hifigan_csmsc_onnx-zh",
    "hifigan_ljspeech_onnx-en",
    "hifigan_aishell3_onnx-zh",
    "hifigan_vctk_onnx-en",
    "hifigan_male_onnx-zh"
]

AM跟VOC需要搭配使用：
- 语言要一致（zh、en、mix、canton），例如am中有-mix的，voc也应该选择-mix的。
- 男音色跟女音色要一致，不然输出声音很怪，例如am中有-male的，那voc也应该选择-male的

你使用am（fastspeech2_male）同时支持zh、en、mix，你的voc也应该选择音色为支持mix的，例如：hifigan_male

**Reply by Samge0** [reply]
上面列出的am_info 跟voc_info 列表，模型名为_onnx前面的内容，_onnx-xx，表示支持的类型/支持的语言类型
